server/bot/imports/tools.ts

```diff
@@ -61,7 +61,7 @@ export const getChannel = (message: Message, arg: string) => {
 // Fresh insults. They come and go, I suppose.
 export const getInsult = () => {
   const insults = [
-    'pathetic lifeform', 'ungrateful bastard', 'idiotic slimeball', 'worthless ass', 'dumb dolt'
+    'pathetic lifeform', 'ungrateful bastard', 'idiotic slimeball', 'worthless ass', 'dumb dolt', 'one pronged fork', 'withered oak', 'two pump chump', 'oompa loompa'
```

Comment:
Hmm not sure what to think of this but these should go on the second line because the line length is too long.

Reply:
```suggestion
    'pathetic lifeform', 'ungrateful bastard', 'idiotic slimeball', 'worthless ass', 'dumb dolt',
    'one pronged fork', 'withered oak', 'two pump chump', 'oompa loompa'
```